java.jdbc-ballerina/Package.md

```diff
@@ -1,96 +1,17 @@
-## Package overview
+## Package Overview
 
-This package provides the functionality required to access and manipulate data stored in any type of relational database 
-that is accessible via Java Database Connectivity (JDBC). 
+The JDBC library is one of the external library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language. This external package is maintained by the ballerina team.
 
-**Prerequisite:** Add the JDBC driver corresponding to the database you are trying to interact with
-as a native library dependency in your Ballerina project. Then, once you build the project by executing the `ballerina build`
-command, you should be able to run the resultant by executing the `ballerina run` command.
+It provides the functionality required to access and manipulate data stored in any type of relational database
+that is accessible via Java Database Connectivity (JDBC).
 
-E.g., The `Ballerina.toml` content.
-Change the path to the JDBC driver appropriately.
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).
```

Comment:
Let's remove BBE link

Reply:
removed the link in https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/160/commits/b31d7e1f1d89c6699dc44d4e3883ee76b35277cf